nms/app/fbcnms-projects/magmalte/docker-compose.yml

```diff
@@ -45,7 +45,7 @@ services:
     environment:
       API_CERT_FILENAME: /run/secrets/api_cert
       API_PRIVATE_KEY_FILENAME: /run/secrets/api_key
-      API_HOST: ${API_HOST:-proxy:9443}
```

Comment:
This block already means use `proxy:9443` if `API_HOST` is undefined https://docs.docker.com/compose/compose-file/#variable-substitution

Since we spin this up in the orc8r docker network, this should work out of the box right?

Reply:
No it doesn't work out of the box. Maybe something else is going on with service names because when I checked the env vars in the container previously, it is `proxy:9443`. However in the NMS all you get is "Cannot communicate with Orchestrator controller" and similar errors.

Reply:
Oh wait didn't the proxy pod get removed recently? I don't see it when I spin up the compose file. This is probably the reason